src/scrapers/leumi-card.js

```diff
@@ -151,10 +150,62 @@ async function getAccountNumber(page, cardIndex) {
   return accountNumber;
 }
 
+async function getChargedCurrencySymbolOfSection(page, cardSection) {
+  const chargedCurrencyElement = await cardSection.$('tbody:first-child > tr:first-child > th:nth-child(7) > a');
+
+  const currencyHeaderValue = await page.evaluate((a) => {
+    return a.innerText;
+  }, chargedCurrencyElement);
+  return currencyHeaderValue.charAt(10);
+}
+
+function parseAmount(amountStr) {
+  if (typeof amountStr === 'number') {
+    return amountStr;
+  }
+
+  if (typeof amountStr === 'undefined' || amountStr === null ||
+    (typeof amountStr === 'string' && amountStr.trim().length === 0)) {
+    return null;
+  }
+
+  const formattedAmount = amountStr
+    .replace(SHEKEL_CURRENCY_SYMBOL, '')
+    .replace(DOLLAR_CURRENCY_SYMBOL, '')
+    .replace(',', '')
+    .replace(/[ ]{2,}/g, ' ')
+    .trim();
+  let currency = SHEKEL_CURRENCY;
+  let amount = null;
+  const parts = formattedAmount.split(' ');
+
+  amount = parseFloat(parts[0]);
+  if (parts.length === 2) {
+    currency = fromCurrencySymbolToValue(parts[1]);
+
+    if (currency === null) {
+      throw new Error(`cannot parse amount, failed to detect currency for '${amountStr}'`);
```

Comment:
@esakal we will probably get a lot of txns in currencies other than ILS or USD.
It feels a bit aggressive to throw an error for such an expected input

Reply:
well I agree it is n aggressive approach. We actually have a softer alternative. if the scraper fails to detect the currency it can set an empty value. That way if someone is interested in the currency symbol and is using different currency will open an issue and others will not be affected.

This is a reasonable choice, but it is  based no the assumption/decision that failing to extract the currency is acceptable and should fallback to empty value.

any thoughts?

Reply:
Hi @eshaham, 
What do you think?

Reply:
@esakal yes, that sounds like a good solution.